docs/sources/community/maintaining/release/1-create-release-branch.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Create Release Branch
+description: Create Release Branch
+---
+# Create Release Branch
+
+A single release branch is created for every `major` or `minor` release (not for patch releases). That release
```

Comment:
Hmm. My understanding here is we have to maintain two different branches for major and minor versions.

e.g: 2.9.x -> 3.0 (will create branch `release-3.0.x`) and all the patch release goes there.
e.g: 3.0.x -> 3.1 (will create branch `realease-3.1.x`) and all the patch release goes there.

Meaning we maintain two different branch `release-3.0.x` and `release-3.1.x` anyway.

I want to mention `every major or minor` (instead of just `every minor`) because that's more intuitive IMHO.